tutorials/strapi-app-serverless-containers-sqldb/index.mdx

```diff
@@ -0,0 +1,833 @@
+        deploy = true
+            
+        environment_variables = {
+            "DATABASE_CLIENT"="postgres",
+            "DATABASE_USERNAME" = scaleway_iam_application.app.id,
+            "DATABASE_HOST" = trimsuffix(trimprefix(regex(":\\/\\/.*:",scaleway_sdb_sql_database.database.endpoint), "://"),":")
+            "DATABASE_NAME" = scaleway_sdb_sql_database.database.name,
+            "DATABASE_PORT" = trimprefix(regex(":[0-9]{1,5}",scaleway_sdb_sql_database.database.endpoint), ":"),
+            "DATABASE_SSL" = "true",
+            "ADMIN_EMAIL" = "${var.ADMIN_EMAIL}"
+        }
+        secret_environment_variables = {
+            "DATABASE_PASSWORD" = scaleway_iam_api_key.api_key.secret_key,
+            "ADMIN_PASSWORD" = "${var.ADMIN_PASSWORD}",
+            "APP_KEYS" = random_bytes.generated_secrets["app_keys"].base64,
+            "API_TOKEN_SALT" = random_bytes.generated_secrets["api_token_salt"].base64,
+            "ADMIN_JWT_SECRET" = random_bytes.generated_secrets["admin_jwt_secret"].base64,
+            "TRANSFER_TOKEN_SALT" = random_bytes.generated_secrets["transfer_token_salt"].base64,
+            "JWT_SECRET" = random_bytes.generated_secrets["jwt_secret"].base64
+        }
+    }
+    
+    resource scaleway_iam_application "app" {
+        name = "tutorial-strapi-blog-tf"
+    }
+    
+    resource scaleway_iam_policy "db_access" {
+        name = "tutorial-strapi-policy-tf"
+        description = "Gives tutorial Strapi blog access to Serverless SQL Database"
+        application_id = scaleway_iam_application.app.id
+        rule {
+            project_ids = ["${var.DEFAULT_PROJECT_ID}"]
+            permission_set_names = ["ServerlessSQLDatabaseReadWrite"]
+        }
+    }
+    
+    resource scaleway_iam_api_key "api_key" {
+        application_id = scaleway_iam_application.app.id
+    }
+    
+    resource scaleway_sdb_sql_database "database" {
+        name = "tutorial-strapi-tf"
+        min_cpu = 0
+        max_cpu = 8
+    }
+    
+    output "database_connection_string" {
+        // Output as an example, you can give this string to your application
+        value = format("postgres://%s:%s@%s",
+            scaleway_iam_application.app.id,
+            scaleway_iam_api_key.api_key.secret_key,
+            trimprefix(scaleway_sdb_sql_database.database.endpoint, "postgres://"),
+        )
+        sensitive = true
+    }
+    
+    output "container_url" {
+        // Output as an example, you can give this string to your application
+        value = scaleway_container.main.domain_name
+        sensitive = true
+    }
+    ```
+
+The Terraform file creates several resources:
+
+- A [Serverless Containers namespace](/serverless/containers/concepts/#namespace), that contains a [Serverless Container](/serverless/containers/quickstart/), which hosts your Strapi application
+
+- A [Serverless SQL Database](/serverless/sql-databases/reference-content/serverless-sql-databases-overview/) which stores posts data
```

Comment:
I don't understand what you mean by "posts data", is it data from blog posts?

Reply:
yes !